packages/react-native-codegen/src/parsers/typescript/parser.js

```diff
@@ -532,6 +532,33 @@ class TypeScriptParser implements Parser {
       extendsProps,
     };
   }
+
+  getProperties(typeName: string, types: TypeDeclarationMap): $FlowFixMe {
```

Comment:
Does this have to be a class member? It doesn't depend on `this` (which is suspicious, btw).